### DISCUSSION TITLE
Classroom assignment forking - Feedback Pull Request

### DISCUSSION THREAD

**Discussion by beak85** [community/community#102022] [GitHub Education]
### Hi everyone,

With the new features enabled on one of my classrooms, I'm not seeing a Feedback Pull Request on the repos after they are forked. I didn't see a notification of that change. I still have the box checked in the assignment settings. Am I missing something?

**Comment by beak85** [comment]
Anyone?

**Reply by AngelaBi** [reply]
I have the same issue. I was wondering if it was related to testing the new feature (forking). I am able to force the pull request by going back to the assignment (after my students fork the assignment), edit, and save  (still with the pull request option checked)

**Reply by bkuker** [reply]
You can force it? My assignment is missing most, but not all, of the feedback PRs and nothing I do changes it

**Comment by walters954** [comment]
This is happening to my classroom as well.

**Comment by mrfabroa** [comment]
Yes, I'm seeing this too.  Some are missing the Feedback PR.  Also, for some that did get the Feedback PR, only the README.md is captured.  Changes to this file are the addition of the "Open in Visual Studio Code" button, however, changes from student work do not get captured in the PR

**Reply by bkuker** [reply]
"Changes from student work"

How are they submitting? I've noticed that certain Classroom automations only run when you push from the ide, NOT when you do an edit directly on the GitHub UI. Are the students editing via the UI on the GitHub website?

NOTE: If this is the case I think there are 2 bugs: feedback not always happening, and separately workflows only triggered by push.

**Reply by mrfabroa** [reply]
Changes are pushed via VS Code.  In some cases, these changes are not included in the feedback PR.

**Reply by bkuker** [reply]
That feels "odd".... It appears to just create a branch named feedback and open a PR from main to feedback... From there on out it's just normal GitHub behavior of any commits added to main being part of the PR. Are you sure they are pushing to main?

**Reply by AngelaBi** [reply]
I am very confused about this. Some projects have the feedback pull request others not, my students all use VS code.  I tried to create a pull request but that would need to be done student by student and I don't want to do that. For me the problems have started since I signed up to use the new upcoming method which  is forking from the starter repository. Maybe there are still bugs since it will be official in June?

**Reply by mrfabroa** [reply]
Odd indeed @bkuker .  I can verify that changes were made to main.   To elaborate on the odd behaviour - I edited the Classroom assignment details (changed the due date) and and after some investigation, it appears that the ones that originally did not get a Feedback PR now have one, and these are the cases where only the README is part of the PR and not the student work commits.
